### PR TITLE
Extend D3M dss outline informations

### PR DESCRIPTION
**Pull request by EmilMalanczak**
fixes FRO-923 FRO-922

### CODE REVIEW

File: packages/app/src/locales/en.po [hunk 1]
```diff
@@ -6,6 +6,12 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: @lingui/cli\n"
 "Language: en\n"
+"Project-Id-Version: \n"
```
Comment (krzkaczor) [inline packages/app/src/locales/en.po:9]: any idea what generated this change? i thought we uploaded up-to-date en.po files few PRs back. If now, please just remove this change.

File: packages/app/src/domain/d3m-info/D3MInfoQuery.ts [hunk 2]
```diff
@@ -56,6 +56,10 @@ export function D3MInfoQuery({ wagmiConfig, chainId }: D3MInfoQueryParams) {
     return {
       D3MCurrentDebtUSD,
       maxDebtCeiling,
+      gap: NormalizedUnitNumber(fromRad(bigNumberify(gap))),
+      increaseCooldown: ttl,
+      lastUpdateBlock: last,
+      lastIncreaseTime: lastInc,
```
Comment (krzkaczor) [inline packages/app/src/domain/d3m-info/D3MInfoQuery.ts:62]:
```suggestion
      lastIncreaseTimestamp: lastInc,
```

File: packages/app/src/domain/d3m-info/D3MInfoQuery.ts [hunk 3]
```diff
@@ -31,7 +31,7 @@ export function D3MInfoQuery({ wagmiConfig, chainId }: D3MInfoQueryParams) {
   const sparkIlkId = stringToHex('DIRECT-SPARK-DAI', { size: 32 })
 
   async function queryFn(): Promise<D3MInfo> {
-    const [[vatArt, vatRate], [IAMLine]] = await multicall(wagmiConfig, {
+    const [[vatArt, vatRate], [IAMLine, gap, ttl, last, lastInc]] = await multicall(wagmiConfig, {
```
Comment (yivlad) [inline packages/app/src/domain/d3m-info/D3MInfoQuery.ts:34]: I'd also rename `IAMLine` to `max`

File: packages/app/src/features/market-details/components/market-overview/DaiMarketOverview.tsx [hunk 4]
```diff
@@ -42,24 +47,39 @@ export function DaiMarketOverview({
...
+            <DetailsGridItem.Title>Utilization rate</DetailsGridItem.Title>
+            <DetailsGridItem.Value>{formatPercentage(utilizationRate)}</DetailsGridItem.Value>
+          </DetailsGridItem>
+
+          <DetailsGridItem>
+            <DetailsGridItem.Title variant="green">Instantly available</DetailsGridItem.Title>
+            <DetailsGridItem.Value>
+              {token.formatUSD(instantlyAvailable, { compact: true })}
+
+              <CooldownTimer
```
Comment (yivlad) [inline packages/app/src/features/market-details/components/market-overview/DaiMarketOverview.tsx:73]: I think the copy in the cooldown should be changed - this timer does not denote caps.

File: packages/app/src/features/market-details/components/status-panel/components/info-tile/InfoTile.tsx [hunk 5]
```diff
@@ -7,7 +7,7 @@ interface InfoTileProps {
 }
 export function InfoTile({ children }: InfoTileProps) {
   return (
-    <div className='grid grid-cols-2 content-start justify-between gap-1.5 sm:grid-cols-1 sm:gap-0.5'>{children}</div>
+    <div className="grid grid-cols-2 content-start justify-between gap-1.5 sm:grid-cols-1 sm:gap-0.5">{children}</div>
```
[the remark this answers is not in the document]
Comment (oskarvu) [reply packages/app/src/features/market-details/components/status-panel/components/info-tile/InfoTile.tsx:10]: Linter works as it should. https://github.com/marsfoundation/spark-app/pull/239/commits/000177a8817118b6d6787e765f5d51a45c7a1243

File: packages/app/src/features/market-details/components/market-overview/DaiMarketOverview.tsx (hunk 4 above)
Comment (EmilMalanczak) [reply packages/app/src/features/market-details/components/market-overview/DaiMarketOverview.tsx:73]: Asked @krzkaczor about that and he said the we don't have to change it right now

**Review by oskarvu — APPROVED** [review]

File: packages/app/src/locales/en.po (hunk 1 above)
Comment (EmilMalanczak) [reply packages/app/src/locales/en.po:9]: Explained on the call

**Review by yivlad — APPROVED** [review]

**Review by krzkaczor — APPROVED** [review]

**Pull request merged**